db-async-common/src/main/java/com/github/jasync/sql/db/Configuration.kt

```diff
@@ -63,7 +65,8 @@ class Configuration @JvmOverloads constructor(
     val executionContext: Executor = ExecutorServiceUtils.CommonPool,
     val currentSchema: String? = null,
     val socketPath: String? = null,
-    val credentialsProvider: CredentialsProvider? = null
+    val credentialsProvider: CredentialsProvider? = null,
+    val rsaPublicKey: Path? = null,
```

Comment:
sorry for asking just now (didn't thought about it before). shouldn't rsaPublicKey be part of  SSLConfiguration? I mean is there a usage for the key without ssl?

Reply:
I had it there initially, but I think it makes more sense to be separate. Since you can decide not to use SSL (use unsafe connection), in which case you need RSA to encrypt the password. So the RSA public key does not imply SSL or the other way around. But if you prefer, I can move it to `SSLConfiguration`

Reply:
if they are independent, I am ok with leaving it as is.